### ISSUE TITLE
Automatic cutting and safe marks

### ISSUE THREAD

**Issue by riverole** [paperize/paperize#9]
**Is your feature request related to a problem? Please describe.**
In Squib, I usually copy and paste templates for the part of the cutting and safe marks. In Paperize, I need to create them every time (when they are usually pretty standard).

**Describe the solution you'd like**
It would be great to have some sort of toggle to automatically have cutting marks on a specific values (that you would be able to modify).

**Describe alternatives you've considered**
I can definitely do that through layers, but this seems a bit cumbersome to do for each component.

**Additional context**
See example from squib:
<img width="589" alt="CuttingAndSafeMarks" src="https://user-images.githubusercontent.com/4833422/73116598-d03c3780-3eed-11ea-8cbf-ef1b63731a16.png">

**Comment by lorennorman** [comment]
Yes! Another thing Paperize used to have in older versions that hasn't made it back, yet.

For this one I'll propose some phases:

1. Simple "Cut-lines" print setting: toggle it on and it draws dotted lines around the components.
2. "Spacing/Thickness" options: do the cut-lines appear over-top of the components, or do they actually push the components apart (possibly wasting paper again)?
3. "Pattern Select": user may not want dotted black/white line, offering a selection of other patterns and coloring options would make this work for all situations

Number 1 should just get done. I'm inclined to wait and let people ask for 2 and 3.

What options does Squib expose?

**Comment by lorennorman** [comment]
Cut lines must also automatically turn off on "component per page" printing mode.
